implement_me.py

```diff
@@ -0,0 +1,115 @@
+from dateutil import parser
+from elasticsearch.helpers import bulk
+from elastic_handler import get_elastic_client
+
+INDEX_NAME = 'entry-index'
+ENTRY_TYPE = 'ip_entry'
+MAX_DOCUMENTS = 100000
+BULK_SIZE = 1000
+
+es = get_elastic_client()
+
+
+def clear_index():
+    es.indices.delete(index=INDEX_NAME, ignore=[400, 404])
+    init_mapping()
+
+
+def init_mapping():
+    index_mapping = """{
```

Comment:
You can use a dict, instead of a giant string.
It'll look much nicer in the IDE, and you'll be able to edit it with Python.

Reply:
Nope, I can't...
the problem is 'true' which is not a reserved known python word and mast be left as is with no hyphens for this to work....
Tried several ways, couldn't make it work.... feel free to try on my station :)